packages/perseus/src/strings.ts

```diff
@@ -1005,14 +1011,17 @@ export const mockStrings: PerseusStrings = {
     srPolygonGraph: "A polygon.",
     srPolygonGraphCoordinatePlane: "A polygon on a coordinate plane.",
     srPolygonGraphPointsNum: ({num}) => `The polygon has ${num} points.`,
+    srPolygonGraphPointsOne: "The polygon has 1 point.",
     srPolygonElementsNum: ({num}) => `A polygon with ${num} points.`,
+    srPolygonElementsOne: "A polygon with 1 point.",
```

Comment:
[Just a comment] It's actually interesting semantically if this is still a "polygon". I think this is the right call given it's specifically a polygon graph, but I thought it was interesting. :)

Reply:
It's definitely weird! I asked about this in the SRUX doc while working on this. I may have to update this in the future depending on the response I get.